sale_order_volume/__openerp__.py

```diff
@@ -19,7 +19,7 @@
 ##############################################################################
```

Comment:
we do some cleaning while we go through module modification could you replace accordingly and add your name https://github.com/coopiteasy/vertical-cooperative/blob/9.0/easy_my_coop/__openerp__.py#L4